test/docs/style/toc-sidebar.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="shrink-to-fit=NO, width=device-width, initial-scale=1.0">
+    <title>Dummy Test</title>
+  </head>
+  <body class="toc-sidebar">
```

Comment:
Shall we add that class to some other existing document, instead of creating a new one just for this?